app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -6,6 +6,15 @@
         orange: "<%= asset_path("logo/logo_orange.png") %>"
       };
     }
+
+    get headshots() {
```

Comment:
@sonia-y not sure you guys need to use react for static pages. Seems overkill and then you have to do gross-ish things like this. Up to you though.